joplin/base/models.py

```diff
@@ -69,14 +69,28 @@ class ServicePage(Page):
 
     vi_panels = [
         # TODO: This field comes from Page and django-modeltranslation complains about it
-        # FieldPanel('title_es'),
+        # FieldPanel('title_vi'),
```

Comment:
❓

Reply:
This was a bad copypasta from the `es_panels` above. This comment keeps it consistent with the other panels by specifying the title field for this panel.

Reply:
gotcha